internal/resource/custom.go

```diff
@@ -48,6 +48,20 @@ func NewCustom(c k8s.Connection, gvr k8s.GVR) *Custom {
 	return cr
 }
 
+func mustExtractMeta(o map[string]interface{}) map[string]interface{} {
+	if m, ok := o["metadata"].(map[string]interface{}); ok {
+		return m
+	}
+	panic("unable to extract meta")
```

Comment:
We should probably take more care with panics and fatals. We should logs these errors/flash and let the view be incomplete. 

It would bad for the user if he wants to use k9s and keep getting panic'd because a single resource contains data that we didn't predicted or because a new version of k8s has changed something.

Reply:
It is ok to let this as a TODO, as long as we can tackle them later.

Reply:
Thank you Gustavo for looking at all these. Did not intend for you to review all these changes as I know there is a boatload here. I think the name here indicates that we expect this call to bomb if things are not as expected aka mustXXX. In this case we're in a constructor context and thus we need to halt as we can't continue if things are not as expected. If the K8s schema does change we would pick this up when we rev the libs which in this case would be a pretty obvious revamp of the resource metadata bloc. That said, In general  though I do agree panic should be reserve only for exceptional situations.